pub-get-report/index.js

```diff
@@ -1,10 +1,11 @@
 const db = require('../database/db');
 
 module.exports = async (context, req) => {
-    const { id } = req.query;
-    const { hash } = req.query;
-    const { url } = req.query;
-    const { testfile } = req.query;
+    const { id, hash, url, testfile, showerrors } = req.query;
+
+    // showErrors - default - true
+    // whether to return the whole errors object in the JSON report response
+    const showErrors = showerrors !== 'false' && showerrors !== 'False';
```

Comment:
```suggestion
    const showErrors = showerrors !== 'false';
```

Reply:
@akmiller01 took a slightly different route, let me know what you think.